src/v/serde/parquet/column_writer.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "serde/parquet/metadata.h"
+#include "serde/parquet/value.h"
+
+namespace serde::parquet {
+
+// A serialized page for a column along with the page header metadata
+// as that is used when creating the metadata for the entire column.
+struct data_page {
+    page_header header;
+    iobuf serialized;
```

Comment:
nit: worth a commnent that the serialized contains a header already ?

Reply:
Yes - done.